.github/workflows/main.yml

```
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
```

Let's keep tests running...
```suggestion
      fail-fast: false
      matrix:
```